refarch-backend/src/main/java/de/muenchen/refarch/configuration/SpringRestConfiguration.java

```diff
@@ -1,5 +1,6 @@
 package de.muenchen.refarch.configuration;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
It's wild that the dependency is from `com.github.spotbugs` but the imports are still totally outdated from FindBugs (predecessor). Well, nothing can be done...